browser/net/brave_site_hacks_network_delegate_helper.cc

```diff
@@ -50,12 +55,91 @@ bool ApplyPotentialReferrerBlock(std::shared_ptr<BraveRequestInfo> ctx) {
   return false;
 }
 
+bool ApplyPotentialQueryStringFilter(std::shared_ptr<BraveRequestInfo> ctx) {
+  DCHECK_CURRENTLY_ON(BrowserThread::IO);
```

Comment:
I don't think this function should care about the exact thread, I think we are safe to remove this check

Reply:
Removed.